src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ArrayCreationAndInitializer.cs

```diff
@@ -435,27 +435,26 @@ public void F(int dimension)
 }
 ";
             string expectedOperationTree = @"
-IArrayCreationOperation (OperationKind.ArrayCreation, Type: System.Int32[], IsInvalid) (Syntax: 'new[2]/*</bind>*/')
```

Comment:
this used to be the case because we'd parse "new [2]" as "new [] { 2 }" (with lots of missing tokens).  The new parse is "new [] { }" (with '2' as skipped text), so we get semantic changes as well.